jp2_online/templates/administracion/search_students.html

```diff
@@ -33,7 +33,7 @@
                   <td>{{student.integrante.get_nivel_estudios_display}}</td>
                   <td>{{student.numero_sae}}</td>
                   <td>
-                    <a class="btn btn-success btn-circle-sm" href="#">
+                    <a class="btn btn-success btn-circle-sm" href="{% url 'administracion:detail_student' student.pk %}">
```

Comment:
i think we should do a big integration test using this link and entering the focus mode once we have that. i don't want to add a test with splinter that simply opens this link.